### ISSUE TITLE
supported tctis not listed

### ISSUE THREAD

**Issue by pohly** [01org/tpm2-tools#602]
https://github.com/01org/tpm2-tools/blob/master/man/common/options.md says:

```
-v, --version: Display version information for this tool, supported tctis and exit.
```

However, I only get the version:
```
# tpm2_dump_capability --version
tpm2_dump_capability, version 2.1.0-dirty
```

I am building tpm2.0-tools rev 27d34a61cb947780ab58508dfea70e696cff185c aka 2.1.0, using meta-measured and refkit/OE-core Pyro. Perhaps the "dirty" suffix comes from the gcc7.diff patch in the recipe.

I can't rule out that my installation is incomplete. The reason I wanted to list supported tctis is this:
```
# tpm2_getrandom -T device:/dev/tpm0
invalid TCTI, see --help
```

**Comment by pohly** [comment]
"tpm2_getrandom -T device" works. I've filed #603 about that.

**Comment by martinezjavier** [comment]
@pohly there were some changes in the tpm2-tools options handling that are not backward compatible with the old version.

You are looking at the options man page for the master branch but using the 2.X branch. Instead you should check the [man page](https://github.com/01org/tpm2-tools/blob/2.X/man/common-options.troff) for that branch (is still in troff, not markdown).

In the old 2.X branch, the -v option only prints the version number, not the available TCTIs.

So basically what you need is something like this for what you want:

```
4 tpm2_getrandom -T device -d /dev/tpm0
```

**Comment by pohly** [comment]
Right, I was looking at the wrong documentation initially. Later I started cross-referencing against the actual documentation for 2.1.0, but hadn't noticed this particular difference. Thanks for pointing it out.
